tests/Common/BranchEventsTest.php

```diff
@@ -240,6 +242,22 @@ public function testFilterBranchTokenEventsIsNotInDefaultBranch(): void
         $this->assertSame($event['id'], $branchEvents[1]['id']);
     }
 
+    /**
+     * @see \Keboola\Test\Utils\EventTesterUtils::initEvents inspired by
+     */
+    private function getLastEventId(Client $client): string
+    {
+        $fireEvent = (new Event())
+            ->setComponent('dummy')
+            ->setMessage('dummy');
+        $lastEvent = $this->createAndWaitForEvent($fireEvent, $client);
+
+        if (!empty($lastEvent)) {
+            return $lastEvent['id'];
+        }
+        $this->fail('Get last event failed - not created');
+    }
+
```

Comment:
Nešlo tohle implementovat v tom event testerutils, než kód kopírovat sem?

Reply:
Tam se do toho motal jeste `tokenId`. Potom neslo pouzit to `listEvents`, protoze se tam pouziva jiny endpoint...